addon/templates/components/uni-footer-website.hbs

```diff
@@ -0,0 +1,3 @@
+<div class="uni-footer-website__wrapper">
+    {{yield (hash row=(component "uni-footer-website-row"))}}
```

Comment:
don't hardcode it here.  Have it defined as a variable in uni-footer-website. 

That way you can redefine it